examples/pytorch/vision/Conference_Room_M4/train.py

```diff
@@ -0,0 +1,249 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
This file seems almost same as Face_Detection/train.py. Please consolidate in to one file.